consent-ws/src/main/java/org/broadinstitute/consent/http/models/Election.java

```diff
@@ -81,6 +112,31 @@ public Election(Integer electionId, String electionType,
         this.finalVoteDate = finalVoteDate;
         this.useRestriction = useRestriction;
         this.translatedUseRestriction = translatedUseRestriction;
+        this.dataUseLetter = dataUseLetter;
+        this.dulName = dulName;
+    }
+
+    public Election(Integer electionId, String electionType,
+            Boolean finalVote, String finalRationale, String status, Date createDate,
+            Date finalVoteDate, String referenceId, Date lastUpdate, Boolean finalAccessVote,
+            UseRestriction useRestriction, String translatedUseRestriction,
+            String dataUseLetter, String dulName, Integer version, Boolean archived) {
+        this.electionId = electionId;
+        this.electionType = electionType;
+        this.status = status;
+        this.createDate = createDate;
+        this.referenceId = referenceId;
+        this.lastUpdate = lastUpdate;
+        this.finalAccessVote = finalAccessVote;
+        this.finalVote = finalVote;
+        this.finalRationale = finalRationale;
+        this.finalVoteDate = finalVoteDate;
+        this.useRestriction = useRestriction;
+        this.translatedUseRestriction = translatedUseRestriction;
+        this.dataUseLetter = dataUseLetter;
+        this.dulName = dulName;
+        this.version = version;
+        this.archived = archived;
     }
```

Comment:
Not necessary for this PR - but something to think about.
you can have constructors invoke each other using this(...) if they take subsets of parameters. Also, for setting the extra values, you can use the setter methods (this way all setting is done via the methods) :

public A(String param1, String param2) {
  setParam1(param1);
  setParam2(param2);
}

public A(String param1, String param2, String param3): this(param1, param2) {
  setParam3(param3);
}